WK2-1852 How interest is paid (#99)
wealthkernel/api-documentation-guides: docs/cash-savings/Interest.md

---
tags: [Cash Savings]
---

# Interest

## How interest is paid

> Interest is paid on the 1st of each month on an accrual basis. This means that interest is calculated (accrued) daily, but paid monthly. 

Reviewer: IMO we don't need to explain accrual basis.
```suggestion
Interest is paid on the 1st of each month on an accrual basis.
```

## Interest rates

> Interest rates can be configured per product, split between the investor and the tenant, up to a maximum of the current [Bank of England base rate](https://www.bankofengland.co.uk/monetary-policy/the-interest-rate-bank-rate).

Reviewer: Should we tweak this to indicate tenants should speak to TMT to configure interest rate?

## Rate changes

> Sometimes the Bank of England changes the base rate. When this happens we will use the most up to date figure and calculate the accrual for that day onwards accordingly. 

Author: This bit is hard to be specific on, as I've no idea what are plans are, e.g. when the rate goes down, do we decrease the tenant share, investor share, neither, both, it depends? Same when the rate goes up.

Reviewer: We should just say that rates will change inline with BoE base rate, I think that's how Griffin word it on their docs.

## Cut offs

> An internal transfer must have been requested before the cut off in order to be eligible for interest. After this time interest will not be accrued until the next working day.

Author: Don't know if a cut off has been decided yet but I'm not sure if we'd put this on the docs anyway.

> An internal transfer must have been requested before the cut off in order to be eligible for interest. After this time interest will not be accrued until the next working day.

Reviewer: ```suggestion
Internal transfers must be requested before [insert cut off time] to be eligible for current day's interest accrual. After this time interest will not be accrued until the next working day.
```

Author: I've done the rewording except for the cut off time, we don't seem to document cut offs publicly in general (and we don't have one for this)

## Withdrawals

> If cash is moved out of the savings portfolio, or completely withdrawn from the platform, interest will be paid as usual on the 1st of the next month, accrued for the days which it was held in the savings portfolio.
> 
> For example, if £100 was held in a savings portfolio from 2025-01-01, and a transfer back to the wallet completes on 2025-01-15, 14 days worth of interest would be paid into the savings portfolio on 2025-02-01.

Reviewer: ```suggestion
When cash is withdrawn from a savings portfolio interest will be paid as usual on the 1st of the next month, accrued for the days which it was held in the savings portfolio.

For example, if £100 was held in a savings portfolio from 2025-01-01, and a transfer back to the wallet completes on 2025-01-15, 14 days worth of interest would be paid into the savings portfolio on 2025-02-01.
```

Author: I worded it in this way because I didn't want to cause confusion as you can't withdraw from a savings portfolio, only transfer cash out. I could just omit the withdrawal part and have something like:
`If cash is transferred out of the savings portfolio, interest earned will be paid as usual on the 1st of the next month`?

## Fractional interest

> If a fractional amount of interest is accrued, i.e. not whole pennies, we will carry the interest forward to the next day. Similarly, if the total accrued interest at the end of the month is fractional, we will store that fraction and add it into next month's payout.

Reviewer: Should we reword this to say we will round interest to the nearest penny each month and any fractional remainder is rolled over?
